File: app/src/main/java/tourtle/ticketing2/model/WisataModel.java
package tourtle.ticketing2.model;

public class WisataModel {
    String idWisata,alamat,idKota,namaWisata;
    Integer parkirBus,parkirRodaDua,parkirRodaEmpat,tiketDomestik,tiketManca,tiketWeekend;

    public WisataModel(String idWisata, String alamat, String idKota, String namaWisata, Integer parkirBus, Integer parkirRodaDua, Integer parkirRodaEmpat, Integer tiketDomestik, Integer tiketManca, Integer tiketWeekend) {
        this.idWisata = idWisata;
        this.alamat = alamat;
        this.idKota = idKota;
        this.namaWisata = namaWisata;
        this.parkirBus = parkirBus;
        this.parkirRodaDua = parkirRodaDua;
        this.parkirRodaEmpat = parkirRodaEmpat;
        this.tiketDomestik = tiketDomestik;
        this.tiketManca = tiketManca;
        this.tiketWeekend = tiketWeekend;
    }

    public String getIdWisata() {
        return idWisata;
    }

    public void setIdWisata(String idWisata) {
        this.idWisata = idWisata;
    }

    public String getAlamat() {
        return alamat;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }

    public String getIdKota() {
        return idKota;
    }

    public void setIdKota(String idKota) {
        this.idKota = idKota;
    }

    public String getNamaWisata() {
        return namaWisata;
    }

    public void setNamaWisata(String namaWisata) {
        this.namaWisata = namaWisata;
    }

    public Integer getParkirBus() {
        return parkirBus;
    }

    public void setParkirBus(Integer parkirBus) {
        this.parkirBus = parkirBus;
    }

    public Integer getParkirRodaDua() {
        return parkirRodaDua;
    }

    public void setParkirRodaDua(Integer parkirRodaDua) {
        this.parkirRodaDua = parkirRodaDua;
    }

    public Integer getParkirRodaEmpat() {
        return parkirRodaEmpat;
    }

    public void setParkirRodaEmpat(Integer parkirRodaEmpat) {
        this.parkirRodaEmpat = parkirRodaEmpat;
    }

    public Integer getTiketDomestik() {
        return tiketDomestik;
    }

    public void setTiketDomestik(Integer tiketDomestik) {
        this.tiketDomestik = tiketDomestik;
    }

    public Integer getTiketManca() {
        return tiketManca;
    }

    public void setTiketManca(Integer tiketManca) {
        this.tiketManca = tiketManca;
    }

    public Integer getTiketWeekend() {
        return tiketWeekend;
    }

    public void setTiketWeekend(Integer tiketWeekend) {
        this.tiketWeekend = tiketWeekend;
    }
}

File: app/src/main/java/tourtle/ticketing2/model/WisataModel.kt
package tourtle.ticketing2.model

class WisataModel(
        private var idWisata: String?,
        private var alamat: String?,
        private var idKota: String?,
        private var namaWisata: String?,
        private var parkirBus: Int?,
        private var parkirRodaDua: Int?,
        private var parkirRodaEmpat: Int?,
        private var tiketDomestik: Int?,
        private var tiketManca: Int?,
        private var tiketWeekend: Int?
) {
    fun getIdWisata(): String? {
        return idWisata
    }

    fun setIdWisata(idWisata: String?) {
        this.idWisata = idWisata
    }

    fun getAlamat(): String? {
        return alamat
    }

    fun setAlamat(alamat: String?) {
        this.alamat = alamat
    }

    fun getIdKota(): String? {
        return idKota
    }

    fun setIdKota(idKota: String?) {
        this.idKota = idKota
    }

    fun getNamaWisata(): String? {
        return namaWisata
    }

    fun setNamaWisata(namaWisata: String?) {
        this.namaWisata = namaWisata
    }

    fun getParkirBus(): Int? {
        return parkirBus
    }

    fun setParkirBus(parkirBus: Int?) {
        this.parkirBus = parkirBus
    }

    fun getParkirRodaDua(): Int? {
        return parkirRodaDua
    }

    fun setParkirRodaDua(parkirRodaDua: Int?) {
        this.parkirRodaDua = parkirRodaDua
    }

    fun getParkirRodaEmpat(): Int? {
        return parkirRodaEmpat
    }

    fun setParkirRodaEmpat(parkirRodaEmpat: Int?) {
        this.parkirRodaEmpat = parkirRodaEmpat
    }

    fun getTiketDomestik(): Int? {
        return tiketDomestik
    }

    fun setTiketDomestik(tiketDomestik: Int?) {
        this.tiketDomestik = tiketDomestik
    }

    fun getTiketManca(): Int? {
        return tiketManca
    }

    fun setTiketManca(tiketManca: Int?) {
        this.tiketManca = tiketManca
    }

    fun getTiketWeekend(): Int? {
        return tiketWeekend
    }

    fun setTiketWeekend(tiketWeekend: Int?) {
        this.tiketWeekend = tiketWeekend
    }

}